### DISCUSSION TITLE
Loading all list items stucked in infinite loop

### DISCUSSION THREAD

**Discussion by Nevinia210** [pnp/pnpcore#724] [Q&A]
Hello Everyone,

yesterday i used version 1.5.17-nightly of PnP.Core for loading all list items with code storageList.Items.ToList(); and it worked like a charm, today im getting stucked in infinite loop of loading items with top/skip options that are increasing in requesting url: 
items?$top=100&$skip=400
is there some version of pnp.core (i tried latest, 1.5 and 1.5.17) or some approach that will allow me to load all list items?

thank you for your answer in advance

**Comment by jansenbe** [comment]
@Nevinia210 : I think I fixed that early this year (see https://github.com/pnp/pnpcore/commit/406b38d495784ea7a675ce57341a0dce4b87975e#diff-645734ff1f2baf8201824157ba680139e052e42ddc315818581bc913a8965060). I would recommend upgrading the latest nightly and check if the issue still happens.
